docs/misc/creating-enrichments.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: Creating Enrichments
+nav_order: 1
+parent: Miscellaneous information
+---
+
+# Adding an enrichment to a collection
+
+## create a collection and add documents to it
+1. Create a new Project, and select the project type to be *document retrieval*
+2. Next, select *Upload data*
+3. Name the collection _Collection1_ and create the collection
+4. Next upload the sample document in this repo at _examples/discovery-search-app/sample-files/watson-in-the-game.pdf_
+5. Select done and wait for this PDF to be ingested, then move on to creating an enrichment
+
+## create an enrichment
+1. Select the project you like to create an enrichment in
+2. Select the *Teach domain concepts* dropdown and then choose *Dictionaries*
+3. If there is not already a dicitonary enrichment, select *Upload*
```

Comment:
Do we need "If there is not already a dictionary enrichment" on this line? Unless the flow to create a new dictionary enrichment changes based on this, I don't think a pre-existing enrichment should affect someone walking through these steps.

Also, typo on "dictionary." Ironic, lol.